src/odbc/src/statement.cpp

```diff
@@ -366,17 +373,11 @@ SqlResult::Type Statement::InternalSetAttribute(int attr, void* value,
       break;
     }
 
+    case SQL_ATTR_PARAM_BIND_OFFSET_PTR:
```

Comment:
This isn't related to parameters. Setting the param bind offset pointer should not affect the row bind offset ptr.

Reply:
I have changed this to simply setting the `ard->GetHeader().bindOffsetPtr`, as described in the [ODBC specification](https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/sqlsetstmtattr-function?view=sql-server-ver16#statement-attributes):
> Setting this statement attribute sets the SQL_DESC_BIND_OFFSET_PTR field in the APD header.

It also says that `SQL_DESC_DATA_PTR`, `SQL_DESC_INDICATOR_PTR`, and `SQL_DESC_OCTET_LENGTH_PTR` should be set but we determined that we won't be doing that.

Reply:
The spec is referring to the APD (Application Parameter Descriptor) which this driver doesn't really implement yet. The APD is for binding parameters for the application to push to the database through the driver.

You're setting it on the ARD (Application Row Descriptor) which is for binding buffers to write query results from SQL fetch.

Reply:
I see, my bad. I have changed this to set `apdi->GetHeader().bindOffsetPtr`, even though the driver doesn't currently use APD.